joblib/pool.py

```diff
@@ -93,6 +93,63 @@ def _get_backing_memmap(a):
         return _get_backing_memmap(b)
 
 
+def _get_temp_dir(pool_folder_name, temp_folder=None):
+    """Get the full path to a subfolder inside the temporary folder.
+
+    This function is originally used in joblib.pool.MemmapingPool, but
```

Comment:
I would remove this paragraph, I am afraid that commenting about history and future potential use cases that may interest 1% of users in the very best case scenario, really belongs in a docstring ...